osf/management/commands/populate_collection_provider_notification_subscriptions.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+from django.contrib.auth.models import Group
+from django.core.management.base import BaseCommand
+from osf.models import NotificationSubscription, CollectionProvider
+
+logger = logging.getLogger(__file__)
+
+
+def populate_collection_provider_notification_subscriptions():
```

Comment:
This has to be run in order to add moderators

Reply:
Technically just so they get emails :P 
Also, make sure you add CollectionProvider to the post_save signals [here]( https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/provider.py#L438) so that new Providers get these by default